src/main/java/com/raoulvdberge/refinedstorage/tile/TileExporter.java

```diff
@@ -59,7 +80,22 @@ public void updateNode() {
                         ItemStack slot = itemFilters.getStackInSlot(i);
 
                         if (!slot.isEmpty()) {
-                            ItemStack took = network.extractItem(slot, upgrades.getItemInteractCount(), compare, true);
+                            int stackSize = upgrades.getItemInteractCount();
+
+                            if (regulator) {
+                                for (int index = 0; i < handler.getSlots(); i++) {
+                                    ItemStack handlerStack = handler.getStackInSlot(index);
+                                    if (API.instance().getComparer().isEqualNoQuantity(slot, handlerStack)) {
```

Comment:
Actually, you might wanna use isEqual(slot, handlerStack, compare) here instead.